js/lessonfilter.js

```diff
@@ -60,13 +99,57 @@ function wireButtons() {
   console.log(uri.toString());
 
   var options = {
-    valueNames: [ 'date', 'title', 'difficulty', 'activity', 'topics' ]
+    valueNames: [ 'date', 'title', 'difficulty', 'activity', 'topics','abstract', 'content' ]
   };
 
   var featureList = new List('lesson-list', options);
   // We need a stateObj for adjusting the URIs on button clicks, but the value is moot for now; could be useful for future functionality.
   var stateObj = { foo: "bar" };
 
+  // Get search indices and corpuses. Right now only trying on English
+  let idx;
+  let corpus;
+  $.getJSON("https://raw.githubusercontent.com/programminghistorian/search-index/master/indices/indexEN.json").done(response => {
```

Comment:
small thing, but pointing to the published github-pages version of this json at https://programminghistorian.github.io/search-index/indices/indexEN.json gives us the benefits of GH Pages' CDN caching. Always prefer that to pointing to raw.githubusercontent.com